src/Domain/Domain.hpp

```diff
@@ -34,10 +34,12 @@ class CoordinateMapBase;
  *  \brief A wrapper around a vector of Blocks that represent the computational
  * domain.
  */
-template <size_t VolumeDim, typename TargetFrame>
+template <size_t VolumeDim>
 class Domain {
  public:
-  explicit Domain(std::vector<Block<VolumeDim, TargetFrame>> blocks) noexcept;
+  using TargetFrame = Frame::Inertial;
```

Comment:
I would remove this in the hpp and just use `Frame::Inertial` in the cpp instead of `TargetFrame` to eliminate references to `TargetFrame` in Domain files